open-apparel-registry/open-apparel-registry
# Pin React dependencies

## Overview

This PR pins the React app's dependencies to specific versions to ensure developer builds are using the same versions of libraries.

Connects #21 

## Notes

The peer dependency warnings described in #12 are still there when installing, but the project will otherwise build successfully.

## Testing Instructions

Set up the project again from this branch and verify that the React app's dependencies install correctly.
